.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+
+python:
+  - "2.6"
+  - "2.7"
+
+script:
+  - python runtests.py
+
+install:
+- pip install django-dirtyfields
```

Comment:
I think you want 
```yaml
install:
  - pip install -e .` here
```
here instead because otherwise we're testing what's on Pypi, not what's in the repository.